sfe/sfe.go

```diff
@@ -0,0 +1,317 @@
+package sfe
+
+import (
+	"embed"
+	"errors"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"strconv"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/go-jose/go-jose/v4"
+	"github.com/go-jose/go-jose/v4/jwt"
+	"github.com/jmhodges/clock"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+
+	"github.com/letsencrypt/boulder/core"
+
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/metrics/measured_http"
+	rapb "github.com/letsencrypt/boulder/ra/proto"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+)
+
+const (
+	// The API version should be checked when parsing parameters to quickly deny
+	// a client request. Can be used to mass-invalidate URLs. Must be
+	// concatenated with other path slugs.
+	unpauseAPISlug = "/sfe/v1"
```

Comment:
nit: I wouldn't call this a "slug". I think of a slug as being the blob at the end of a URL, like an account or order's unique ID. This is a (leading!) path component. Maybe "unpauseAPIPrefix"?